File: solutions/swift/raindrops/1/Sources/Raindrops/Raindrops.swift
final class Raindrops {
    let number: Int

    init(_ number: Int) {
        self.number = number
    }

    var sounds: String {
        var result = ""
        if number.isMultiple(of: 3) {
            result += "Pling"
        }
        if number.isMultiple(of: 5) {
            result += "Plang"
        }
        if number.isMultiple(of: 7) {
            result += "Plong"
        }
        return result.isEmpty ? "\(number)" : result
    }
}

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "fmt"

func Convert(input int) string {
	var result = ""
	if input%3 == 0 {
		result += "Pling"
	}
	if input%5 == 0 {
		result += "Plang"
	}
	if input%7 == 0 {
		result += "Plong"
	}
	if len(result) == 0 {
		return fmt.Sprint(input)
	}
	return result
}
